packages/lib/internal/fetcher.ts

```diff
@@ -35,6 +35,8 @@ export type FetcherOptions = {
 export type RequestOptions = {
   // External signal for when to abort the request.
   abortSignal?: AbortSignal;
+  // A custom host override. Intended only for usage with self-hosted storage.
+  hostOverride?: string;
```

Comment:
Similar to `headers` that is available in both here and `FetcherOptions`, no need for the `override` suffix.

```suggestion
  // Optionally override the host used in this request. Takes precedence over the fetcher's host.
  //
  // For example, this can be used to send a request to a self-hosted storage zone instead of the main Airplane API.
  host?: string;
```

Reply:
Fixed.